pom.xml

```diff
@@ -167,6 +167,11 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>okhttp</artifactId>
+			<version>4.12.0</version>
```

Comment:
@benliscmp can we just use the existing one?

Reply:
Removed okhttp 4.12.0 in pom